### PR TITLE
Adds three new ear sprites

### PR DESCRIPTION
**Pull request by Z0w0plankton**
## About The Pull Request

Adds three new choices for ears, each habing four ears; A version of the teshari upright for regular body shapes, a version of the teshari mane for regular body shapes, and an original third sprite. 

## How This Contributes To The Skyrat Roleplay Experience

More options for ears, no longer will teshari hoard all the four-eared sprites.

## Proof of Testing

dreamseeker_buRH4nNtX2
dreamseeker_ti9AZJtidn
dreamseeker_Zwgv2gkeSk

## Changelog

:cl:
add: Three new choices for ears, all of which have four ears.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/ears.dm [hunk 1]
```diff
@@ -350,3 +350,12 @@
 /datum/sprite_accessory/ears/mutant/mouse_two
 	name = "Mouse II"
 	icon_state = "mouse_two"
+/datum/sprite_accessory/ears/mutant/big/temp3
```
Comment (Zonespace27) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/ears.dm]: Add spacing like is shown above this

**Review by Zonespace27 — DISMISSED** [review]

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/ears.dm (hunk 1 above)
Comment (Z0w0plankton) [reply modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/ears.dm]: Should be done!

**Comment by Kepteyn** [comment]
I like the look of these, I'm getting ideas for how these could be used just looking at them.

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/ears.dm [hunk 2]
```diff
@@ -350,3 +350,15 @@
...
+
+/datum/sprite_accessory/ears/mutant/big/temp3
+	name = "Four Ears 1"
+	icon_state = "four_ears_1"
+
+/datum/sprite_accessory/ears/mutant/temp
+	name = "Four Ears 2"
+	icon_state = "four_ears_2"
+
+/datum/sprite_accessory/ears/mutant/big/temp2
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/ears.dm]: temp...

**Review by tf-4 — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/ears.dm (hunk 2 above)
Comment (Z0w0plankton) [reply modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/ears.dm]: Replaced with proper names now

**Review by tf-4 — APPROVED** [review]

**Review by Ebin-Halcyon — APPROVED** [review]

**Pull request merged**